knative-operator/pkg/controller/knativeserving/consoleutil/util.go

```diff
@@ -0,0 +1,32 @@
+package consoleutil
```

Comment:
When we add new optional components we can rename this to `capabilities` and move it to some other structure.

Reply:
+1 for capabilities package (might be a follow up)